### PR TITLE
PLF-6595: Top navigation should be fixed

### PR DESCRIPTION
**Pull request by boueslati**
Fix Top Toolbar when scrolling to the bottom of the page

### CODE REVIEW

File: platform-ui-skin/src/main/webapp/skin/less/platform/skin/platform.less [hunk 1]
```diff
@@ -22,6 +22,10 @@
 	background: @contentBackground;
 }
 
+body .RightBodyTDContainer, body .LeftNavigationTDContainer {
+    padding-top: 42px;
```
Comment (thomasdelhomenie) [inline platform-ui-skin/src/main/webapp/skin/less/platform/skin/platform.less]: I think you should use @navbarHeight instead.

Comment (boueslati) [inline platform-ui-skin/src/main/webapp/skin/less/platform/skin/platform.less]: Done.
Currently @navbarHeight is not applied to UIToolbarContainer. We will have to merge this [PR](https://github.com/exoplatform/platform-ui/pull/52)

Comment (thomasdelhomenie) [inline platform-ui-skin/src/main/webapp/skin/less/platform/skin/platform.less]: Yes I agree. We are waiting for the CLA of the contributor.
But it is not blocker for this PR.

**Comment by thomasdelhomenie** [comment]
LGTM

**Comment by thomasdelhomenie** [comment]
Merged

**Pull request closed without merging**